account/src/main/resources/org/killbill/billing/account/dao/AccountEmailSqlDao.sql.stg

```diff
@@ -38,9 +38,9 @@ and t.is_active
 >>
 
 markEmailAsDeleted() ::= <<
-update <tableName()> t
-set t.is_active = 0
-where <idField("t.")> = :id
-<AND_CHECK_TENANT("t.")>
+update <tableName()>
```

Comment:
I agree the `t.` was not really needed (since we don't do any join), so it is ok to remove it. However is that because postgresql syntax does not support that?